build.rs

```diff
@@ -102,3 +105,26 @@ fn get_java_codegen_output_directory() -> PathBuf {
     }
     path.to_path_buf()
 }
+
+fn expand_equality_macro(out: &str) {
```

Comment:
Since it would be a silent bug for us to generate `equals` but forget to create `hashCode` in Java, would it make sense to generate them both together here? Essentially this would just mean moving
```    
method hash(&self) -> i32; alias hashCode; 
```
into this function.

Reply:
That's a good point. I was trying to reduce how much code had to be expanded, but you're right that we should make sure that they're both present, and the best way to do that is to expand them both. I'll make that change.

Reply:
Done